thinc/layers/expand_window.py

```diff
@@ -1,31 +1,58 @@
-from typing import Tuple, Callable
+from typing import Tuple, TypeVar, Callable, Union, cast
 
 from ..model import Model
 from ..config import registry
-from ..types import Floats2d
+from ..types import Floats2d, Ragged
 
 
-InT = Floats2d
-OutT = Floats2d
+InOutT = TypeVar("InOutT", Floats2d, Ragged)
```

Comment:
In these cases we'd just call the type `InT`, like we do for `residual` for instance, we'd just have `Model[InT, InT]`.

Reply:
Fixed.